app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/dataset_editing/TakePictureActivityTest.kt

```diff
@@ -92,7 +107,10 @@ class TakePictureActivityTest {
         onView(withText("cat3")).check(matches(isDisplayed()))
     }
 
+
+
     @Test
+    @Ignore // haven't found a way without FragmentScenario, which doesn't seem to be possible with Hilt
```

Comment:
```suggestion
    @Ignore // Comparison of arguments in action failed even though they seem to be the same. Suspecting it is because of DummyCategory's equals()
```

If I understand correctly

Reply:
This was about fragment scenario but it is fixed now. The 2 tests with the comparison failing are in AddPictureActivityTest